styleguide/screens/Patterns/Tabs/Tabs.test.js

```diff
@@ -3,6 +3,11 @@ import { render } from 'react-dom';
 
 import Tabs from './Tabs';
 
+jest.mock('react-router-dom', () => ({
+  // eslint-disable-next-line react/prop-types
+  Link: ({ children }) => <span>{ children }</span>,
```

Comment:
Do you know why it's necessary to mock this?

Reply:
Using a `Link` outside of a router will throw undefined context error. We have also done the same in https://github.com/appearhere/bloom/blob/master/styleguide/components/Navigation/Navigation.test.js